### ISSUE TITLE
Object.dirname is deprecated.

### ISSUE THREAD

**Issue by btito** [baldurh/atom-status-bar-blame#1]
Argument to `path.dirname` must be a string
```
Object.dirname (/Applications/Atom.app/Contents/Resources/app.asar/src/electron-shims.js:9:10)
Object.git (/Users/btito/.atom/packages/atom-status-bar-blame/node_modules/blamer/src/vcs/git.js:22:18)
Blamer.blameByFile (/Users/btito/.atom/packages/atom-status-bar-blame/node_modules/blamer/src/Blamer.js:15:32)
Object.<anonymous> (/Users/btito/.atom/packages/atom-status-bar-blame/lib/utils/blame.js:22:31)
undefined.next (null:null:null)
step (/Users/btito/.atom/packages/atom-status-bar-blame/lib/utils/blame.js:7:273)
```

**Comment by baldurh** [comment]
Thanks for the heads up! Seems to be an issue when you create a new file in an empty window. Right?

**Comment by btito** [comment]
Hi,

I think it is when you edit a file outside a git repo. I noticed this when editing styles.less

Met vriendelijke groet,
Bob Tito

**Comment by baldurh** [comment]
Fixed by 5d61ee488c813c9d7bbb793d0e13e244d1c6a136
